docs/low-code-sql/data-tests/use-project-tests.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Use project tests
+id: use-project-tests
+description: Custom SQL query tests
+sidebar_position: 8
+tags:
+  - how-to
+  - testing
+  - sql
+---
+
+Project tests are custom data tests that check for a passing condition, such as checking that the given query doesn’t return any unexpected rows. They are based on [dbt singular tests](https://docs.getdbt.com/docs/build/data-tests#singular-data-tests), but you don’t need to know dbt to use project tests in Prophecy. Project tests are useful for data users who know the important business questions but wouldn’t necessarily know how to write tests.
+
+Project tests perform a SQL query that checks the executed result of your project against a passing condition to see if it is successful. This is done by checking whether a particular table, generated by combining one or more database objects from your transformation, meets a certain condition.
```

Comment:
I think instead of "transformation" we should say "Project"

Reply:
Done.